source/error_handler.cc

```diff
@@ -33,11 +33,17 @@ ErrorHandler<ntables>::ErrorHandler (   const std::string name,
 template <int ntables>
 void ErrorHandler<ntables>::declare_parameters (ParameterHandler &prm)
 {
+  std::string dummy_names;
+  for (unsigned int i=0; i<ntables-1; ++i)
+    {
+      dummy_names = dummy_names + "error ,";
+    }
+  dummy_names = dummy_names + "error";
   prm.declare_entry ("Write error files", "false", Patterns::Bool());
   prm.declare_entry ("Output error tables", "true", Patterns::Bool());
   prm.declare_entry ("Error file format", "tex", Patterns::Selection("tex|txt|gpl|org"));
   prm.declare_entry ("Compute error", "true", Patterns::Bool());
-  prm.declare_entry ("Table names", "error", Patterns::List(Patterns::Anything(), 1, ntables),
```

Comment:
Also, the list should be *exactly* ntables long:

~~~
prm.declare_entry ("Table names", "error", Patterns::List(Patterns::Anything(), ntables, ntables),
~~~